src/utils/money.cljs

```diff
@@ -247,6 +247,3 @@
   [:=> [:cat [:maybe :int]]
    [:maybe :string]])
 
-(defn amount-in-hex
```

Comment:
not sure about this, but the money utils are quite generic and `amount-in-hex` seems more specific to the wallet. So I moved it to utils there, but happy to leave it as is if preferred 👍

Reply:
Send is such a large area I would prefer to keep some utils there that are specific to send related parts!

Reply:
I also prefer to have some naming to utils once they grow too much, not a big fan of lots of random functionality in a utils file, we should also keep an eye on that imo